docs/services/frame-system/_index.md

```diff
@@ -115,11 +115,125 @@ The resulting tree of reference frames looks like:
 
 ## Accessing the Frame System
 
-The [Robot API](https://github.com/viamrobotics/api/blob/main/proto/viam/robot/v1/robot.proto) supplies two methods to interact with the Frame System through gRPC calls:
+The [Robot API](https://github.com/viamrobotics/api/blob/main/proto/viam/robot/v1/robot.proto) supplies two methods to interact with the Frame System:
 
-1. `TransformPose`: Transforms a pose measured in one reference frame to the same pose as it would have been measured in another.
-2. `FrameSystemConfig`: Returns a topologically sorted list of all the reference frames monitored by the frame system.
-Any [supplemental transforms](#supplemental-transforms) are also merged into the tree, topologically sorted, and returned.
+| Method Name | Description |
+| ----- | ----------- |
+<!-- | [`FrameSystemConfig`](#framesystemconfig) | Return a topologically sorted list of all the reference frames monitored by the frame system. | -->
+| [`TransformPose`](#transformpose) | Transform a pose measured in one reference frame to the same pose as it would have been measured in another. |
```

Comment:
```suggestion
| [`TransformPose`](#transformpose) | Transform a pose measured in one reference frame to the same pose as it would have been measured in another. |
<!-- | [`FrameSystemConfig`](#framesystemconfig) | Return a topologically sorted list of all the reference frames monitored by the frame system. | -->
```